sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -102,16 +102,11 @@ export class StreamingReceiver extends MessageReceiver {
    * @param {ReceiveOptions} [options]     Receive options.
    * @return {Promise<StreamingReceiver>} A promise that resolves with an instance of StreamingReceiver.
```

Comment:
The docs for return type need to be updated

Reply:
Done